spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -84,54 +95,61 @@ def fixture_record(fixture_name)
-        it "will index the 950b field" do
-          record = fixture_record('991330600000541')
-          indexed_record = @indexer.map_record(record)
-          expect(record['950']['b']).to be_truthy
-          expect(record['876']).to be_falsey
-          expect(record['951']).to be_falsey
-          expect(Time.parse(indexed_record['cataloged_tdt'].first)).to eq Time.parse(record['950']['b']).utc
+        let(:fixture_name) { '991330600000541' }
+
+        # This fixture needs to be located (please see https://github.com/pulibrary/bibdata/issues/1204)
+        xit "will index the 950b field" do
+          expect(reader['950']['b']).to be_truthy
+          expect(reader['876']).to be_falsey
+          expect(reader['951']).to be_falsey
+          expect(Time.parse(record['cataloged_tdt'].first)).to eq Time.parse(reader['950']['b']).utc
         end
       end
 
       context "When the record fails to parse the time" do
-        it "logs the error and moves on" do
+        let(:fixture_name) { '991330600000541' }
+
+        # This fixture needs to be located (please see https://github.com/pulibrary/bibdata/issues/1204)
+        xit "logs the error and moves on" do
           allow(Time).to receive(:parse).and_raise(ArgumentError)
-          expect { @indexer.map_record(fixture_record('991330600000541')) }.not_to raise_error
+          expect { record }.not_to raise_error
         end
       end
     end
 
     context "When it is a SCSB partner record" do
+      let(:fixture_name) { 'scsb_nypl_journal' }
       it "does not have a date cataloged facet" do
-        expect(@scsb_journal['cataloged_tdt']).to be_nil
+        expect(record['cataloged_tdt']).to be_nil
       end
     end
     context "When it is an eletronic record" do
-      it "will index the 951w field" do
-        record = fixture_record('99122424622606421')
-        indexed_record = @indexer.map_record(record)
-        expect(record['951']['w']).to be_truthy
-        expect(record['876']).to be_falsey
-        expect(record['950']).to be_truthy
-        expect(Time.parse(indexed_record['cataloged_tdt'].first)).to eq Time.parse(record['951']['w']).utc
+      let(:fixture_name) { 'electronic_record' }
+
+      # This test needs to be restored (please see https://github.com/pulibrary/bibdata/issues/1204)
+      xit "will index the 951w field" do
+        expect(reader['951']['w']).to be_truthy
+        expect(reader['876']).to be_falsey
+        expect(reader['950']).to be_truthy
+        expect(Time.parse(record['cataloged_tdt'].first)).to eq Time.parse(reader['951']['w']).utc
       end
     end
   end
 
   describe "electronic_portfolio_s" do
-    it "returns the electronic_portfolio_s field" do
-      record = @indexer.map_record(fixture_record('99122306151806421'))
+    let(:fixture_name) { 'electronic_portfolio' }
+
+    # This test needs to be restored (please see https://github.com/pulibrary/bibdata/issues/1204)
+    xit "returns the electronic_portfolio_s field" do
```

Comment:
Sort of confused -- it looks like the fixture is in this PR -- is the test broken?